pubsub/helpers_test.go

```diff
@@ -0,0 +1,30 @@
+package pubsub_test
+
+import (
+	"fmt"
+	"runtime/debug"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/tendermint/tmlibs/pubsub/query"
+)
+
+func assertReceive(t *testing.T, expected interface{}, ch chan interface{}, msgAndArgs ...interface{}) {
+	select {
+	case actual := <-ch:
+		assert.Equal(t, expected, actual, msgAndArgs)
+	case <-time.After(10 * time.Millisecond):
```

Comment:
might be tight! maybe a time.Second is better ? Just thinking about CI servers and VMs ...